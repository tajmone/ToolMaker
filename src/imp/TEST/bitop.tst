%%ESET VARIABEL(-1 BITAND 256)
%%(VARIABEL)
%%ESET VARIABEL(5 BITOR 7)
%%(VARIABEL)
%%((5-1) BITAND (1+3))
%%(5-1 BITAND 1+3)
%%BEGIN(bittest(num))
%%(num)
%%END(bittest)
%%EINCLUDE(bittest(3 BITOR 6))
