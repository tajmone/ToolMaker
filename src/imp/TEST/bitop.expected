########## bitop ##########
256
7
4
3
7
